src/pages/Menu.tsx

```diff
@@ -1,5 +1,10 @@
 const Menu = () => {
-	return <h2>Menu</h2>
+	return (
+		<>
```

Comment:
Why do we need a fragment here?

Reply:
Forgot to remove while testing the table - called its component here.

Already cleaned up